usr/share/doc/onion-grater-merger/examples/40_bisq.yml

```diff
@@ -60,8 +60,10 @@
       response:
         - pattern:     '650 HS_DESC CREATED (\S+) (\S+) (\S+) \S+ (.+)'
           replacement: '650 HS_DESC CREATED {} {} {} redacted {}'
-        - pattern:     '650 HS_DESC UPLOAD (\S+) (\S+) .*'
-          replacement: '650 HS_DESC UPLOAD {} {} redacted redacted'
+        - pattern:     '650 HS_DESC FAILED (\S+) \S+ \S+ \S+ (.+)'
+          replacement: '650 HS_DESC FAILED {} redacted redacted redacted {}'
+        - pattern:     '650 HS_DESC UPLOAD (\S+) (\S+) (\S+) (\S+) (\S+)'
+          replacement: '650 HS_DESC UPLOAD {} {} {} {} {}'
```

Comment:
Are all those fields in `HS_DESC UPLOAD` required to be seen by the Bisq client?

[Cwtch can use less information](https://github.com/Whonix/onion-grater/blob/master/usr/share/doc/onion-grater-merger/examples/40_cwtch.yml#L71-L72) and [Onionshare also](https://github.com/Whonix/onion-grater/blob/master/usr/share/doc/onion-grater-merger/examples/40_onionshare.yml#L89-L90).